<launch>
    <node pkg="turtlesim" exec="turtlesim_node"/>
    <node pkg="celestial_turtle_hunter" exec="turtle_hunter_node">
        <param name="spawn_position_x" value="5.544445"/>
        <param name="spawn_position_y" value="1.0"/>
        <param name="spawn_orientation" value="1.5708"/>
    </node>
</launch>